src/commands/emulators-start.ts

```diff
@@ -95,22 +95,23 @@ function printEmulatorOverview(options: any): void {
       .map((emulator) => {
         const emulatorName = Constants.description(emulator).replace(/ emulator/i, "");
         const isSupportedByUi = EMULATORS_SUPPORTED_BY_UI.includes(emulator);
-        // The Extensions emulator runs as part of the Functions emulator, so display the Functions emulators info instead.
-        const info = EmulatorRegistry.getInfo(emulator);
+        const info = EmulatorRegistry.getRawInfo(emulator);
         if (!info) {
           return [emulatorName, "Failed to initialize (see above)", "", ""];
         }
+        let uiLink = "n/a";
+        if (isSupportedByUi && uiRunning) {
+          const url = EmulatorRegistry.url(Emulators.UI);
+          url.pathname = `/${emulator}`;
+          uiLink = stylizeLink(url.toString());
+        }
 
-        return [
-          emulatorName,
-          EmulatorRegistry.getInfoHostString(info),
-          isSupportedByUi && uiInfo ? stylizeLink(`${uiUrl}/${emulator}`) : clc.blackBright("n/a"),
-        ];
+        return [emulatorName, info.host, info.port, uiLink];
```

Comment:
I'm certain the reason we had the full links before was so that they were clickable from a terminal - I remember it being a specific request! Could we maintain that?

Reply:
Good catch! I've restored the behavior. Added some comments too for background.